Client/Frontend/Onboarding/ViewControllers/OnboardingDefaultSettingsViewController.swift

```diff
@@ -0,0 +1,255 @@
+            scrollView.frameLayoutGuide.bottomAnchor.constraint(equalTo: view.bottomAnchor, constant: -UX.scrollViewVerticalPadding),
+            scrollView.frameLayoutGuide.heightAnchor.constraint(equalTo: containerView.heightAnchor).priority(.defaultLow),
+
+            scrollView.contentLayoutGuide.leadingAnchor.constraint(equalTo: contentContainerView.leadingAnchor),
+            scrollView.contentLayoutGuide.topAnchor.constraint(equalTo: contentContainerView.topAnchor),
+            scrollView.contentLayoutGuide.trailingAnchor.constraint(equalTo: contentContainerView.trailingAnchor),
+            scrollView.contentLayoutGuide.bottomAnchor.constraint(equalTo: contentContainerView.bottomAnchor),
+            scrollView.contentLayoutGuide.widthAnchor.constraint(equalTo: scrollView.frameLayoutGuide.widthAnchor),
+
+            // Content view wrapper around text
+            contentContainerView.topAnchor.constraint(equalTo: containerView.topAnchor),
+            contentContainerView.leadingAnchor.constraint(equalTo: containerView.leadingAnchor),
+            contentContainerView.bottomAnchor.constraint(equalTo: containerView.bottomAnchor),
+            contentContainerView.trailingAnchor.constraint(equalTo: containerView.trailingAnchor),
+
+            contentStackView.topAnchor.constraint(greaterThanOrEqualTo: contentContainerView.topAnchor, constant: topPadding),
+            contentStackView.leadingAnchor.constraint(equalTo: contentContainerView.leadingAnchor, constant: leadingPadding),
+            contentStackView.trailingAnchor.constraint(equalTo: contentContainerView.trailingAnchor, constant: -trailingPadding),
+            contentStackView.bottomAnchor.constraint(greaterThanOrEqualTo: contentContainerView.bottomAnchor, constant: -bottomPadding),
+            textStackView.leadingAnchor.constraint(equalTo: contentStackView.leadingAnchor),
+            primaryButton.leadingAnchor.constraint(equalTo: contentStackView.leadingAnchor),
+            primaryButton.trailingAnchor.constraint(equalTo: contentStackView.trailingAnchor),
+            contentViewHeightConstraint
+        ])
+    }
+
+    private func updateLayout() {
+        titleLabel.text = viewModel.title
+        primaryButton.setTitle(viewModel.buttonTitle, for: .normal)
+    }
+
+    private func addViewsToView() {
+        createLabels(viewModel.descriptionSteps)
+
+        contentStackView.addArrangedSubview(titleLabel)
+        numeratedLabels.forEach { textStackView.addArrangedSubview($0)}
+        contentStackView.addArrangedSubview(textStackView)
+        contentStackView.addArrangedSubview(primaryButton)
+
+        containerView.addSubview(contentStackView)
+        contentContainerView.addSubview(containerView)
+        scrollView.addSubview(contentContainerView)
+        view.addSubview(scrollView)
+
+        view.backgroundColor = .white
+    }
+
+    private func createLabels(_ descriptionTexts: [String]) {
```

Comment:
```suggestion
    private func createLabels(from descriptionTexts: [String]) {
```
nit: reads better when invoking